WIP events/React docs (#13965)
PrefectHQ/prefect: docs/3.0rc/react/incidents.mdx

---
title: Incidents
description: Incidents in Prefect Cloud help identify, resolve, and document issues in mission-critical workflows.
---

> Incidents are a Prefect Cloud feature to help your team manage workflow disruptions. Incidents help you identify, resolve, and document issues with mission-critical workflows. This system enhances operational efficiency by automating the incident management process and providing a centralized platform for collaboration and compliance.

Reviewer: ```suggestion
Incidents help you manage inevitable workflow disruptions, minimize their impact, and ensure operational continuity. With incidents, you can identify, resolve, and document issues with workflows, facilitating collaboration and compliance.
```

> ## Definition

Reviewer: ```suggestion
```

> Incidents are formal declarations of disruptions to a workspace. With [automations](#incident-automations), activity in a workspace can be paused when an incident is created and resumed when the incident is resolved.

Reviewer: ```suggestion
Incidents vary in nature and severity, ranging from minor glitches to critical system failures. With [automations](#incident-automations), activity in a workspace can be paused when an incident is created and resumed when the incident is resolved.
```

> Incidents vary in nature and severity, ranging from minor glitches to critical system failures. Prefect Cloud enables users to effectively and automatically track and manage these incidents, ensuring minimal impact on operational continuity.

Reviewer: ```suggestion
```

> ![Incidents in the Prefect Cloud UI](/3.0rc/img/ui/incidents-dashboard.png)

Reviewer: ```suggestion
```

> ## Use cases

Reviewer: ```suggestion
```

> Incorporating incidents into your Prefect workflow facilitates many common needs:

Reviewer: ```suggestion
Incorporating incidents into your Prefect workflow facilitates:
```

* **Comprehensive impact assessment**, Users gain insights into the incident's influence on workflows, helping in prioritizing response efforts

* **Compliance with incident management processes**, Detailed documentation and reporting features support compliance with incident management systems

* **Enhanced operational transparency**, The system provides a transparent view of both ongoing and resolved incidents, promoting accountability and continuous improvement

## Incident management in Prefect Cloud

Prefect Cloud provides tools for managing the entire incident lifecycle, from creation to retrospective reporting. 

### Create an incident

There are several ways to create an incident:

* **From the Incidents page**, click the **+** button, fill in required fields, and attach any Prefect resources related to your incident.

* **From a flow run, work pool, or block**, from the failed flow run, click the menu button and select "Declare an incident". This method automatically links the resource. 

* **Through an [automation](/3.0rc/react/understand-events/automations-triggers/)**, set up incident creation as an automated response to selected triggers.

### Automate an incident

> Automations can be used for triggering an incident and for selecting actions to take when an incident is triggered. For example, a work pool status change could trigger the declaration of an incident, or a critical level incident could trigger a notification action.

Reviewer: ```suggestion
Automations can be used for triggering an incident and for taking action when an incident is triggered. For example, a work pool status change could trigger the declaration of an incident, or a critical level incident could trigger a notification action.
```

> To automatically take action when an incident is declared, set up a custom trigger that listens for declaration events.

Reviewer: ```suggestion
To automatically take action when an incident is declared, set up a custom trigger that listens for declaration events:
```
